src/client/pages/Checkout/shared/CheckoutPageWrapper.tsx

```diff
@@ -44,9 +47,11 @@ const Wrapper = styled.div`
 const InnerWrapper = styled.div`
   margin: 0 auto;
   position: relative;
+  background: ${colorsV3.gray200};
 
-  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+  ${MEDIUM_LARGE_SCREEN_MEDIA_QUERY} {
     padding: 6.5rem 2rem;
+    background: none !important;
```

Comment:
Do we really need to use `!important` here?

Reply:
nope, removed